aviary/utils/process_input_decks.py

```diff
@@ -150,11 +146,27 @@ def update_options(aircraft_values: AviaryValues(), initial_guesses):
     -------
     tuple: Updated aircraft values and initial guesses.
     """
+    GASP_defaults = AviaryValues()
+    GASP_defaults.set_val('INGASP.JENGSZ', val=4)
+    GASP_defaults.set_val('test_mode', val=False)
+    GASP_defaults.set_val('use_surrogates', val=True)
+    GASP_defaults.set_val('mass_defect', val=10000, units='lbm')
+    GASP_defaults.set_val('problem_type', val=ProblemType.SIZING)
+    GASP_defaults.set_val(Aircraft.Electrical.HAS_HYBRID_SYSTEM, val=False)
+    GASP_defaults.set_val(Aircraft.Design.RESERVES, val=4998)
+
+    # overwrite GASP_defaults with values from aircraft_values, then replace
+    # aircraft_values with this merged set
+    GASP_defaults.update(aircraft_values)
+    aircraft_values = GASP_defaults
+
+    for var in initial_guesses:
+        if var in aircraft_values:
+            initial_guesses[var], _ = aircraft_values.get_item(var)
```

Comment:
Is this necessary? the initial guesses should all be distinct names that are separate from those in aircraft_values